extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -121,11 +123,11 @@ private function getSortingParam( $sortingType ) {
 			return [ 'sort' => $sortingType ];
 		}
 
-		return [];
+		return [ ];
 	}
 
 	private function getArticleContributeText( $metadata, $type ) {
-		if ( !empty( $metadata['wantedBy'] ) ) {
+		if ( !empty( $metadata[ 'wantedBy' ] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wantedBy" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)